js/app.jsx

```diff
@@ -486,6 +486,25 @@ var Job = React.createClass({
       </ul>
     );
 
+    function slimTrace(str) {
+      if (!str) {
+        return null;
+      }
+      var linesRegex = /(\w+)\.scala:(\d+)/i;
```

Comment:
Is there a reason to require a `.scala` in there? Won't `/[\w.]+:\d+/i` get the job done?